linchpin/shell/__init__.py

```diff
@@ -336,8 +340,23 @@ def fetch(ctx, fetch_type, remote, root):
               metavar='TX_ID', required=False,
               help="Display a specific transaction by ID (tx_id)."
                    " Only works with '--view=tx'")
+@click.option('--output-format', type=str, default="cfg",
+              metavar='output_format', required=False,
+              help="Inventory output format")
+@click.option('--output-file', type=str,
+              metavar='output_file', required=False,
+              help="inventory output file path")
+@click.option('--output-type', type=str,
+              metavar='output_type', required=False,
+              help="inventory output file path")
```

Comment:
If we genericize this option, we could have `--output-type` be 'inventory', 'pinfile', 'distilled', 'latest', etc. If no `--output-type` is passed, each method should determine the default.

Reply:
@herlo : ^^^ Totally agree with the direction .